README.md

```diff
@@ -42,7 +67,7 @@ Y. Song, S. Guadarrama, K. Murphy_, ["Speed/accuracy trade-offs for modern convo
 5. [Clean Up](#5-clean-up)
```

Comment:
I know you didn't cause this but I noticed this link doesn't point anywhere. Maybe you could fix it with this PR?

Reply:
nice catch, I fixed it